spec/models/ext_management_system_spec.rb

```diff
@@ -18,12 +18,13 @@
 
   let(:all_types_and_descriptions) do
     {
-      "ansible_tower_configuration" => "Ansible Tower Configuration",
+      "ansible_tower_automation"    => "Ansible Tower Automation",
       "azure"                       => "Azure",
       "azure_network"               => "Azure Network",
       "ec2"                         => "Amazon EC2",
       "ec2_network"                 => "Amazon EC2 Network",
       "ec2_ebs_storage"             => "Amazon EBS",
+      "embedded_ansible_automation" => "Embedded Ansible Automation",
```

Comment:
I guess technically no one on the end, but it might show up as a "provider type" in a summary screen or something.

It's not in this PR, but at some point, we will have to exclude it from the supported_emses list so they don't appear in the "create provider" screens.  Perhaps we do that with permissions.yml...haven't quite decided yet.